python/pynessie/model.py

```diff
@@ -276,9 +276,9 @@ def get_obj_type(self, obj: Operation) -> str:
 class CommitMeta:
     """Dataclass for commit metadata."""
 
-    hash_: str = desert.ib(fields.Str(data_key="hash"), default=None)
-    commitTime: datetime = desert.ib(fields.DateTime(), default=None)
-    authorTime: datetime = desert.ib(fields.DateTime(), default=None)
+    hash_: str = attr.ib(default=None, metadata=desert.metadata(fields.Str(data_key="hash")))
```

Comment:
looking at this again wondering we also need `allow_none=True` as we do below? :thinking: 
if we have a test that uses this class without a `hash_` it is fine i guess.

Reply:
This works as well:
```
hash_: str = attr.ib(default=None, metadata=desert.metadata(fields.Str(data_key="hash", allow_none=True)))
```

P.S. Updated PR name and description.